app/code/Magento/Catalog/composer.json

```diff
@@ -31,7 +31,8 @@
         "magento/module-ui": "*",
         "magento/module-url-rewrite": "*",
         "magento/module-widget": "*",
-        "magento/module-wishlist": "*"
+        "magento/module-wishlist": "*",
+        "magento/module-grouped-product": "*"
```

Comment:
@shmVan any specific reason to include `module-grouped-product` in `require`

Reply:
@engcom-Hotel, because the code itself got dependent on this module.
We have open issue for that https://github.com/magento/magento2/issues/37322